packages/guui/README.md

```diff
@@ -0,0 +1,187 @@
+# Guui
+
+A set of guardian-styled components and helpers
+
+**Alpha package - under construction 🚧**
+
+This package is still under heavy construction, please do not install this in applications intended for
+use in production. The API may change significantly before it is out of alpha. If you enjoy working on shifting sands and wish to use this package in your application, please talk to the Platform team, or email dotcom.platform.
+
+## Install
+
+```bash
+$ npm install @guardian/guui
+```
+
+```bash
+$ yarn add @guardian/guui
+```
+
+## Usage
+
+GUUI components are [REACT](https://www.google.com) components that use [EMOTION](https://emotion.sh/docs/introduction) for writing their associated styles in JavaScript (css-in-js).
+
+`guui` contains a library of reusable components that you can import into your own `React` application. It also contains a number of helpers for creating your own `guui` style components and rendering them on the client and the server.
+
+### Prerequisites
+
+When importing or creating a component with `guui` you must ensure `React` is available in your module. Why? because the JSX a component such as:
+
+```js
+<Headline/>
+```
+
+Will be transpiled to:
+
+```js
+React.createElement(Headline)
+```
+
+**How to can you ensure `React` is available?**
+
+**Option 1:** Import your own `React` dependency.
+
+```js
+import React from 'react'
+```
```

Comment:
I don't think consumers should be bringing their own version of React. I can't see any advantage, only potential compatibility issues?

Reply:
The alternative being they use `React` exported by `guui`? I'd really like to get to the bottom of this one. I've read quite a few `README`'s over the past week from React component libraries. I've not seen a single instance where they say you must use a copy of `React` exported by them. If a consumer had an established React project and then wanted to use `guui` components would they need to go through and update any references to `import React from 'react'`? That seems strange. The component library defining a supported version of React as a `peerDependency` seems the most common approach, the library specifies "react": ">=16.4.0" in `peerDependencies`; if the consumer has a compatible version everything is fine, if not they won't be able to install the library.